test/libsinsp_e2e/vtidcollision.c

```diff
@@ -16,7 +16,10 @@ limitations under the License.
 
 */
 
+#ifndef _GNU_SOURCE
```

Comment:
If we don't use `add_definitions(-D_GNU_SOURCE)` above, we can avoid this small patch.

Reply:
Fixed in https://github.com/falcosecurity/libs/pull/2271/commits/4cde899f228bed4b6e68e4ad7df31638047a11ed.